CONFIGDOC.md

```diff
@@ -1,18 +1,18 @@
 You can use the Jenkins&trade; plugin for MATLAB&reg; in freestyle and multi-configuration projects. This guide demonstrates how to run MATLAB code on Jenkins, considering a Microsoft&reg; Windows&reg; platform.
 
--  [Use MATLAB for Build](#use-matlab-for-build)
+-  [Add MATLAB to System Path](#add-matlab-to-system-path)
 -  [Specify Build Steps](#specify-build-steps)
       - [Run MATLAB Tests](#run-matlab-tests)
       - [Run MATLAB Command](#run-matlab-command)
 -  [Set Up Freestyle Project](#set-up-freestyle-project)
 -  [Set Up Multi-Configuration Project](#set-up-multi-configuration-project)
 
-## Use MATLAB for Build
-Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox is displayed in the **Build Environment** section of the project configuration window. Selecting the checkbox instructs Jenkins to use MATLAB for your build.
+## Add MATLAB to System Path
+Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox is displayed in the **Build Environment** section of the project configuration window.
 
 ![matlab_path](https://user-images.githubusercontent.com/48831250/76883191-1c528b00-6852-11ea-9242-7d143f8a5c46.png)
 
-When you select the **Use MATLAB version** checkbox, you can specify the full path to the MATLAB installation folder that is used for the build. The specified directory in the **MATLAB root** box is prepended to the PATH environment variable, and the specified MATLAB version is used to perform the build. You must not leave the **MATLAB root** box blank if no MATLAB directory exists on the path. If you leave the box blank, then the system tries to use the MATLAB directory closest to the top of the system path.
+Select the **Use MATLAB version** checkbox to specify the full path to the MATLAB root folder that should be used for the build. The specified directory in the **MATLAB root** box is prepended to the PATH environment variable, and the matlab.exe file in the bin subdirectory is used to perform the build. You must not leave the **MATLAB root** box blank if no MATLAB directory exists on the path. If you do not select the checkbox or if you leave the text box blank, then the system tries to use the MATLAB directory closest to the top of the system path.

```

Comment:
This is not quite right. The user specifies the MATLAB root (not including the bin subdir). What happens is the root + bin subdir is the ting prepended to the PATH.

So the "specified directory" is the MATLAB root, but the thing that is added to the PATH is the bin dir under the root that is specified. The takeaway is that they specify "where" their MATLAB is and the right thing is added to the PATH.

Also, this seems too strong:
    _You must not leave the **MATLAB root** box blank if no MATLAB directory exists on the path._

It is OK if they leave the box blank. They should just know that the plugin nis going to call "matlab" and if it can't find it the build will fail.

I like the idea of clarifying here in this section that if the build agent already has the desired MATLAB on the PATH then there is no need to check this box at all. That might even be a nice workflow. Whoever sets up the agents ensures MATLAB is available and people configuring builds don't even have to worry about it.

Reply:
Good points. How about the revised text?